includes/wikia/models/NavigationModel.class.php

```diff
@@ -556,16 +522,16 @@ public function parseOneLine($line) {
 			} else {
 				if ( empty( $link ) ) {
 					$href = '#';
-				} else if( $link{0} == '#' ) {
+				} else if ( $link{0} == '#' ) {
 					$href = '#';
 				} else {
 					//BugId:51034 - URL-encoded article titles break inter-wiki links
 					//in global Nav (but work correctly in the article body)
 					$title = Title::newFromText( urldecode( $link ) );
 
-					if ( $title instanceof Title  ) {
+					if ( $title instanceof Title ) {
 						$href = $title->fixSpecialName()->getLocalURL();
-						$pos = strpos($link, '#');
+						$pos = strpos( $link, '#' );
 						if ( $pos !== false ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 4 "if", "for", "while", "switch" and "try" statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS134)